lua/astrocommunity/pack/purescript/init.lua

```diff
@@ -0,0 +1,82 @@
+return {
+  {
+    "srghma/nvimmer-ps",
```

Comment:
Missing opts table.
```suggestion
    "srghma/nvimmer-ps",
    opts = {},
```

Reply:
yes

![2024-11-11_18-38-07](https://github.com/user-attachments/assets/b04ce867-b664-404d-9343-7ffbef3ae1d8)

![2024-11-11_18-38-48](https://github.com/user-attachments/assets/b00a0b32-18fc-468d-a5d9-b9f626ff55aa)

Reply:
I mean these: 
![image](https://github.com/user-attachments/assets/3e6d6e06-51e6-4c9f-84c3-329367339352)

Reply:
Also the purescript docs should really include its own way of installing the packages, not refer to astrocommunity. Thats a bad move.

Reply:
bad move fixed

commands are shown

![2024-11-12_10-19-50](https://github.com/user-attachments/assets/5209cea0-1663-47ed-b6d9-6df9c6d8f54b)